Corona-Warn-App/src/deviceForTesters/res/layout/fragment_test_appconfig.xml

```diff
@@ -14,7 +14,7 @@
         android:paddingBottom="32dp">
 
         <LinearLayout
-            style="@style/card"
+            style="@style/Card"
```

Comment:
just a question, I saw some styles names start with lower case letters and some with capital case letters. 
what is our convention?

Reply:
As far as I can see, there is none. I would prefer them upper case, because GreyCard.Ripple looks better to me.

Reply:
I like that way too.